playbook/app/pb_kits/playbook/data/samples.yml

```diff
@@ -7,3 +7,4 @@ full_page_samples:
 - trending_repositories
 - crowdsourced_posts
 dashboard_cards:
+- card_dashboard
```

Comment:
I renamed the `dashboard_cards` category to `card_samples`.  You'll have to resolve that merge conflict.  `dashboard_cards` probably ought to be the name of the same itself, not the category.

Reply:
Just updated that, let me know if this PR passes for release today 😊